ui/src/pages/version/components/forms/components/ResourcesPanel.js

```diff
@@ -69,7 +68,16 @@ export const ResourcesPanel = ({
     } else {
       setGpuValueOptions([{ value: "None", inputDisplay: "None" }]);
     }
-  }, [resourcesConfig, resourcesConfig.gpu_name, gpus]);
+
+    if (resourcesConfig.min_replica > maxAllowedReplica) {
+      onChange("min_replica")(maxAllowedReplica - 1)
```

Comment:
it should be allowed to set min-max as a fixed number of `maxAllowedReplica`?

Reply:
We aren't stopping the user from setting min_replica=max_replica. This condition only sets min_replica=`maxAllowedReplica - 1` as a default when a user changes the environment with the min_replica value **set higher than** the max allowed value for the new environment.